src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -507,11 +541,11 @@ export default class geTemplateBuilderFormFields extends LightningElement {
     handleDeleteFormElement(event) {
         const fieldMapping = TemplateBuilderService.fieldMappingByDevName[event.detail.fieldName];
         const element = this.template.querySelector(`lightning-input[data-field-mapping="${event.detail.fieldName}"]`);
+
         element.checked = false;
 
-        if (fieldMapping.Is_Required) {
-            this.validate();
-        }
+        this.validate();
```

Comment:
We may need to also call this `validate` in the `handleToggleFieldMapping` method when `fieldMapping.Is_Describable === false` so that the validations are re-triggered when a user removes the broken field mapping through the sidebar.

Reply:
fixed!